### PR TITLE
fix issues caused by update to Atom 1.7.0 (atom issue #11302)

### PR DESCRIPTION
**Pull request by ayan4m1**
This resolves platformio-atom-ide-terminal #12 in Win7/10. It is not the right fix, but it is a viable workaround that can be safely removed later. Thanks, I appreciate your efforts with this project.

### CODE REVIEW

File: lib/platformio-ide-terminal.coffee [hunk 1]
```diff
@@ -68,7 +68,9 @@ module.exports =
           default: do ->
             if process.platform is 'win32'
               path = require 'path'
-              path.resolve(process.env.SystemRoot, 'System32', 'WindowsPowerShell', 'v1.0', 'powershell.exe')
+              # Patch to deal with various casings of this variable in Windows, see atom issue 1337
```
Comment (ayan4m1) [inline lib/platformio-ide-terminal.coffee:71]: 11302

**Comment by ivankravets** [comment]
Thanks! `SystemRoot` is OK, see Dump https://github.com/atom/atom/issues/11460

The problem is in different version of node and node-gyp. We are working on rebuilding pty.js binaries.

**Comment by ayan4m1** [comment]
Yeah, I manually installed node-gyp globally, then ran `apm rebuild` in the dev platformio-ide-terminal directory - that gave me the proper binary.

**Comment by ivankravets** [comment]
The latest release should work without problems.

**Pull request closed without merging**